boards/adi/max32657evkit/doc/index.rst

```diff
@@ -0,0 +1,385 @@
+.. zephyr:board:: max32657evkit
+
+MAX32657
+########
+
```

Comment:
zephyr board directive already takes care of this

```suggestion
```

Reply:
Removed, thanks.